src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class Lottery {
+    public Ball getRandomBall() {
```

Comment:
Lottery class should have Random and ColorSupplier as isntruments to create parameters of the ball